Ambiguous call Encoding.GetByteCount(char[], int, int) and Encoding.GetByteCount(string, int, int)

Seen here https://jenkins.mono-project.com/job/test-mono-acceptancetests/960/

```
external/coreclr/tests/src/CoreMangLib/cti/system/text/unicodeencoding/unicodeencodinggetbytecount1.cs(189,37):
error CS0121: The call is ambiguous between the following methods or properties:
'Encoding.GetByteCount(char[], int, int)' and 'Encoding.GetByteCount(string, int, int)'
```

Seems to have started with https://github.com/mono/mono/pull/7948

--- a/acceptance-tests/SUBMODULES.json
+++ b/acceptance-tests/SUBMODULES.json
@@ -10,7 +10,7 @@
   {
     "name": "coreclr", 
     "url": "git://github.com/mono/coreclr.git", 
-    "rev": "83d8279997d8ce4ad344ff9b937b2d13d074dcaa", 
+    "rev": "aa62c0382ae79562293f3ea1a94ef0f16f80f498", 
     "remote-branch": "origin/mono", 
     "branch": "mono", 
     "directory": "coreclr"

